docs/workflow_generators/residential_hpxml.rst

```diff
@@ -36,41 +36,61 @@ Configuration Example
-
-- ``simulation_output_report``: Update the arguments to the `ReportSimulationOutput`_ measure. See
-  :ref:`sim-output-report-defaults` for current defaults.
+  - ``measure_dir_name``: Name of measure directory.
+  - ``arguments``: map of key, value arguments to pass to the measure.
 
-- ``reporting_measures`` (optional): a list of reporting measures to apply
-  to the workflow. Any columns reported by these additional measures will be
-  appended to the results csv. Note: For upgrade runs, do not add
-  ``ApplyUpgrade`` to the list of reporting measures, doing so will cause run
-  to fail prematurely. ``ApplyUpgrade`` is applied automatically when the
-  ``upgrades`` key is supplied.
+- ``simulation_output_report``: Update the arguments to the `ReportSimulationOutput`_ measure.
+  See :ref:`sim-output-report-defaults` for current defaults.
+
+  - ``timeseries_frequency``: The frequency at which to report timeseries output data. Using 'none' will disable timeseries outputs. Valid choices are 'none', 'timestep', 'hourly', 'daily', and 'monthly'.
+  - ``include_timeseries_total_consumptions``: Generates timeseries energy consumptions for building total.
+  - ``include_timeseries_fuel_consumptions``: Generates timeseries energy consumptions for each fuel type.
+  - ``include_timeseries_end_use_consumptions``: Generates timeseries energy consumptions for each end use.
+  - ``include_timeseries_emissions``: Generates timeseries emissions. Requires the appropriate HPXML inputs to be specified.
+  - ``include_timeseries_hot_water_uses``: Generates timeseries hot water usages for each end use.
+  - ``include_timeseries_total_loads``: Generates timeseries total heating, cooling, and hot water loads.
+  - ``include_timeseries_component_loads``: Generates timeseries heating and cooling loads disaggregated by component type.
+  - ``include_timeseries_zone_temperatures``: Generates timeseries temperatures for each thermal zone. 
+  - ``include_timeseries_airflows``: Generates timeseries airflows.
+  - ``include_timeseries_weather``: Generates timeseries weather data.
+  - ``add_timeseries_dst_column``: Optionally add, in addition to the default local standard Time column, a local clock TimeDST column. Requires that daylight saving time is enabled.
+  - ``add_timeseries_utc_column``: Optionally add, in addition to the default local standard Time column, a local clock TimeUTC column. If the time zone UTC offset is not provided in the HPXML file, the time zone in the EPW header will be used.
+
+- ``reporting_measures`` (optional): A list of reporting measures to apply to the workflow.
+  Any columns reported by these additional measures will be appended to the results csv.
+  Note: For upgrade runs, do not add ``ApplyUpgrade`` to the list of reporting measures, doing so will cause run to fail prematurely.
+  ``ApplyUpgrade`` is applied automatically when the ``upgrades`` key is supplied.
```

Comment:
I think in the past there had been some confusion around this- some people were trying to add `ApplyUpgrade` to the `reporting_measures` block.

Reply:
Can we update bsb to exclude this measure specifically, or more generally exclude any measure that's not a reporting measure, rather than having to document that a user shouldn't do this?

Reply:
Seems like that should happen in a base class (i.e., not this workflow generator specifically).

Reply:
I mean maybe, or at least provide a validation error. There's a couple of tricky things in that `reporting_measures` isn't defined on the base class, but is implemented by each workflow generator class. It seems like a nice to have, but not critical.